src/main/services/connectors.service.ts

```
@@ -565,7 +614,7 @@ export default class ConnectorsService {
       }
       case 'databricks':
         // Use token-based authentication with no username (UID)
-        const TOKEN = `db-token-${projectName}`;
+        const TOKEN = `db-token-${conn.name}`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)